src/modules/completedTask.js

```diff
@@ -0,0 +1,17 @@
+// eslint-disable-next-line
```

Comment:
delete this comment and solve the problem according to the configurations of Linters and it will be perfect.

Reply:
thank you so much for highlighting the issues I will resolve these issues thank you.